src/puppetlabs/puppetdb/jdbc.clj

```diff
@@ -360,44 +360,32 @@
            (throw e)))))))
 
 (defn make-connection-pool
-  "Create a new database connection pool"
-  [{:keys [classname subprotocol subname user username password
-           partition-conn-min partition-conn-max partition-count
-           stats log-statements log-slow-statements statements-cache-size
-           conn-max-age conn-lifetime conn-keep-alive read-only?
-           connection-timeout pool-availability-threshold]
-    :as db}]
-  (let [;; Load the database driver class explicitly, to avoid jar load ordering
-        ;; issues.
-        _ (Class/forName classname)
-        log-slow-statements-duration (pl-time/to-seconds log-slow-statements)
-        config          (doto (new BoneCPConfig)
-                          (.setDefaultAutoCommit false)
-                          (.setLazyInit true)
-                          (.setMinConnectionsPerPartition partition-conn-min)
```

Comment:
These extra configuration items, some are no longer needed, should we throw an error, deprecate & retire etc.? We should be making it clear they aren't going to work at least somewhere. log-slow-statements is an example. I think the partition config was _never_ documented.